src/utils/sendTaskUpdates.ts

```diff
@@ -0,0 +1,24 @@
+import { env } from "../typeDefinitions/default.types";
+import config from "../../config/config";
+
+export async function sendTaskUpdate(
+  completed: string,
+  planned: string,
+  blockers: string,
+  env: env
+): Promise<void> {
+  const formattedString = `**Completed**: ${completed}\n\n**Planned**: ${planned}\n\n**Blockers**: ${blockers}`;
+  const bodyObj = {
+    content: formattedString,
+  };
+  const url = config(env).TRACKING_CHANNEL_URL;
+  await fetch(url, {
+    method: "POST",
+    body: JSON.stringify(bodyObj),
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bot ${env.DISCORD_TOKEN}`,
+    },
+  });
+  return;
```

Comment:
Explicit Return:  Add return Promise.resolve(); at the end to indicate successful execution.

Error Handling: Consider adding error handling (e.g., using a try...catch block) to gracefully handle potential issues during the fetch operation.